.github/actions/setup-js-env/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Setup Node and run npm ci'
+description: "run actions/setup-node and `npm ci`"
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version: '16'
+        cache: 'npm'
+
+    - name: Install Dependencies
+      shell: bash
+      run: npm ci --ignore-scripts
```

Comment:
It is also possible to include `install` `cache` `lint` `test` actions into a compose action, and controls by `inputs` (with `with` keyword)

Reply:
that's too complex I think